### ISSUE TITLE
[Question] How can I disable the logger ?

### ISSUE THREAD

**Issue by Baker68** [ulixee/hero#145]
I got a log of this  : 
```
CaptureWebsocketMessageError.UnregisteredResource {
  event: {
    resourceId: undefined,
    message: 'something',
    isFromServer: true,
    lastCommandId: 4,
    timestamp: 1661695335885.463
  },
  context: { sessionId: '_tShQs1NKdCzPBzVd04dU' },
  sessionId: '_tShQs1NKdCzPBzVd04dU',
  sessionName: 'default-session'
}
```
and some messages are very long ; How can I disabled the logger or `UnregisteredResource` ?

Settings `process.env.DEBUG = 'false';` or `process.env.ULX_DEBUG = 'false'` dose nothing and I am using `disableMitm: true`;

Thanks in advance.

**Comment by blakebyrnes** [comment]
Thanks for reporting. We'll need a code change to disable that message. I hope you've seen in some places, but `disableMitm` is NOT a very stable feature. It was added at the request of a user, but before that was mostly used internally to workaround occasional issues with the MITM or for narrowing down why we were getting block

**Comment by blakebyrnes** [comment]
Fixed in 2.0.0-alpha.11
